test/HtmlUtils-test.tsx

```diff
@@ -60,6 +60,7 @@ describe('HtmlUtils', () => {
         enableHtmlTopicFeature();
         const component = mount(<div>{ topicToHtml("**pizza** 🍕", "<b>pizza</b> 🍕", null, false) }</div>);
         const wrapper = component.render();
-        expect(wrapper.children().first().html()).toEqual("<b>pizza</b> <span class=\"mx_Emoji\" title=\":pizza:\">🍕</span>");
+        expect(wrapper.children().first().html()).toEqual(
+            "<b>pizza</b> <span class=\"mx_Emoji\" title=\":pizza:\">🍕</span>");
```

Comment:
closing bracket needs to be on a new line if we're going to do it this way, which means trailing comma. Alternatively, move the whole `.toEqual(...)` to a new (indented) line.